prefect_aws/client_parameters.py

```diff
@@ -0,0 +1,57 @@
+"""Module handling Client parameters"""
+
+import dataclasses
+from dataclasses import dataclass
+from typing import Any, Dict, Optional, Union
+
+
+@dataclass(frozen=True)
+class AwsClientParameters:
+    """
+    Dataclass used to manage extra parameters that you can pass when you initialize the Client.If you
+    want find more information
+    [boto3 docs](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html)
+    for more info about the possible client configurations.
+
+    Args:
+        :type api_version: Optional[string]
+        :param api_version: The API version to use.  By default, botocore will
```

Comment:
Can the docstrings be re-formatted to conform with Google docstrings:
https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
Done. I updated it. Can you confirm the new format is what you expect?